modules/home/classes/transformations-manager.php

```diff
@@ -15,6 +15,7 @@ class Transformations_Manager {
 		'Create_New_Page_Url',
 		'Filter_Plugins',
 		'Remove_Sidebar_Upgrade_For_Pro_Users',
+		'Filter_Elementor_AI',
```

Comment:
I think it could be improved by storing here class name like `Filter_Elementor_AI::class`. After this you don't need to build a class path in foreach here 
```
foreach ( $transformations as $transformation_id ) {
    $classes[ $transformation_id ] = new $transformation_id( $arguments );
}
```

Reply:
not critical, we can leave it as it is

Reply:
I like it

Reply:
@mykytamurzin How we can include the namespace in this? I think we will get the incorrect namespace in this way.

`Elementor\Modules\Home\Classes` instead of `Elementor\Modules\Home\Transformations`